charts/openshift-metering/templates/presto/presto-coordinator-statefulset.yaml

```diff
@@ -28,6 +28,12 @@ spec:
         presto-common-config-hash: {{ include (print $.Template.BasePath "/presto/presto-common-config.yaml") . | sha256sum }}
         presto-catalog-config-hash: {{ include (print $.Template.BasePath "/presto/presto-catalog-config-secret.yaml") . | sha256sum }}
         presto-jmx-config-hash: {{ include (print $.Template.BasePath "/presto/presto-jmx-config.yaml") . | sha256sum }}
+{{- if .Values.presto.spec.presto.config.tls.enabled }}
```

Comment:
This also requires `createSecret` to be true right? Otherwise we're using the users secret and we won't be using this one so hashing it isn't useful.

maybe `if and .Values.presto.spec.presto.config.tls.enabled .Values.presto.spec.presto.config.tls.createSecret`

Reply:
Yeah, that needed to be updated. There was also a poor copy-paste job done in the reporting-operator deployment that I updated also.